suite/tests/client-interface/file_io.c

```diff
@@ -270,9 +308,32 @@ main()
      */
 #ifdef WINDOWS
     _control87(_MCW_EM & (~_EM_ZERODIVIDE), _MCW_EM);
+#elif defined(LINUX)
+    if (feenableexcept(FE_DIVBYZERO) == -1) {
+#    ifdef AARCH64
+        /* This call may return EPERM on AArch64.
+         * https://code.woboq.org/userspace/glibc/sysdeps/aarch64/fpu/feenablxcpt.c.html#37
```

Comment:
I see the same on https://www.qnx.com/developers/docs/7.0.0/#com.qnx.doc.neutrino.lib_ref/topic/f/feenableexcept.html. Maybe worth adding Assad to understand this limitation better?

Reply:
Also on https://code.woboq.org/userspace/glibc/sysdeps/aarch64/fpu/feenablxcpt.c.html#37 as mentioned in the PR' description.

Reply:
So if the AArch64 hardware doesn't support floating-point traps, then this will return EPERM, and we do not fail the test.  But we don't then have a test on hardware that does have traps; we could then have something untested and not realize it.  Not sure failing the test is good either.  It's similar to rseq or close_range or other things: we manually make sure our test machines have the feature, and we have the tests gracefully skip that if not present.

Reply:
@AssadHashmi Is this something that can be solved by updating the kernel of the Jenkins machine? The first link says it's a limitation of the hardware (unmasking of floating-point exceptions being ignored), which makes me think updating the kernel might not help.

On a related note, there are some syscalls that aren't supported by the current kernel there, notably `close_range`. Not sure if it's easy to update the kernel on that machine. It is 5.4 I think, which is ~1.5 years old.

Reply:
I guess we can follow up offline. Resolving for now.